pkg/mapx/struct.go

```diff
@@ -433,7 +511,7 @@ func (m *MapXStruct) doWriteMap(cfg string, targetValue reflect.Value, sourceMap
 	return nil
 }
 
-func (m *MapXStruct) doWriteSlice(cfg string, targetValue reflect.Value, sourceValues *MapX) error {
+func (m *Struct) doWriteSlice(cfg string, targetValue reflect.Value, sourceValues *MapX) error {
```

Comment:
Maybe you want to switch the receiver name to `s` instead of `m`

Reply:
done